src/main/java/subway/dto/LineRequest.java

```diff
@@ -4,17 +4,19 @@ public class LineRequest {
 
     private String name;
     private String color;
+    private int extraCharge;
     private String upStation;
     private String downStation;
     private int distance;
 
     public LineRequest() {
     }
 
-    public LineRequest(String name, String color, String upStation, String downStation,
+    public LineRequest(String name, String color,int extraCharge,String upStation, String downStation,
         int distance) {
         this.name = name;
         this.color = color;
+        this.extraCharge=extraCharge;
```

Comment:
```suggestion
        this.extraCharge = extraCharge;
```
자동 포맷팅 단축키(`cmd` + `opt` + `l`)를 익혀두시면 좋습니다 😎

Reply:
헉 깜빡하고 누락되었나보네요😂
반영했습니다! 감사합니다ㅎㅎ